tracer/src/Datadog.Trace/ClrProfiler/ServerlessInstrumentation/Serverless.cs

```diff
@@ -0,0 +1,95 @@
+                var serverlessDefinitions = GetServerlessDefinitions();
+                NativeMethods.InitializeProfiler(DefinitionsId, serverlessDefinitions);
+                foreach (var def in serverlessDefinitions)
+                {
+                    def.Dispose();
+                }
+
+                Debug("The profiler has been initialized with serverless definitions, count = " + serverlessDefinitions.Length);
+            }
+        }
+
+        internal static bool IsRunningInLambda(string extensionPath)
+        {
+            string path = EnvironmentHelpers.GetEnvironmentVariable(ExtensionEnvName) ?? extensionPath;
+            return File.Exists(path) && !string.IsNullOrEmpty(EnvironmentHelpers.GetEnvironmentVariable(FunctionEnvame));
+        }
+
+        internal static NativeCallTargetDefinition[] GetServerlessDefinitions()
+        {
+            try
+            {
+                LambdaHandler handler = new LambdaHandler(EnvironmentHelpers.GetEnvironmentVariable(HandlerEnvName));
+                string assemblyName = typeof(InstrumentationDefinitions).Assembly.FullName;
+                string integrationType = GetIntegrationTypeFromParamCount(handler.ParamTypeArray.Length);
+                return new NativeCallTargetDefinition[]
+                {
+                    new(handler.GetAssembly(), handler.GetFullType(), handler.GetMethodName(), handler.ParamTypeArray, 0, 0, 0, 65535, 65535, 65535, assemblyName, integrationType)
+                };
+            }
+            catch (Exception ex)
+            {
+                Serverless.Error("Impossible to get Serverless Definitions : " + ex.Message);
+                return Array.Empty<NativeCallTargetDefinition>();
+            }
+        }
+
+        internal static string GetIntegrationTypeFromParamCount(int paramCount)
+        {
+            int inputParamCount = paramCount - 1; // since the return type is in the array
+            switch (inputParamCount)
+            {
+                case 0:
+                    return typeof(AWS.LambdaNoParam).FullName;
+                case 1:
+                    return typeof(AWS.LambdaOneParam).FullName;
+                case 2:
+                    return typeof(AWS.LambdaTwoParams).FullName;
+                default:
+                    throw new ArgumentOutOfRangeException("AWS Lambda handler number of params can only be 0, 1 or 2.");
+            }
+        }
+
+        internal static void Debug(string str)
+        {
+            if (string.Equals(EnvironmentHelpers.GetEnvironmentVariable(LogLevelEnvName), "debug", StringComparison.OrdinalIgnoreCase))
+            {
+                Console.WriteLine("{0} {1}", DateTime.Now.ToString("yyyy-MM-dd MM:mm:ss:fff"), str);
+            }
+        }
+
+        internal static void Error(string str)
+        {
+            Console.WriteLine("{0} {1}", DateTime.Now.ToString("yyyy-MM-dd MM:mm:ss:fff"), str);
```

Comment:
We probably want UTC time to avoid time-zone issues, and you want `HH` for hour (`MM` is month).
```suggestion
            Console.WriteLine("{0} {1}", DateTime.UtcNow.ToString("yyyy-MM-dd HH:mm:ss:fff"), str);
```